docs/contributing/source/dev-setup-coding-and-contribution-process/run-dev-network-stack.md

```diff
@@ -0,0 +1,296 @@
+# Run a BigchainDB network
+
+**NOT for Production Use**
+
+You can use the following instructions to deploy a single or multi node
+BigchainDB network for dev/test using the extensible `stack` script(s). 
+
+Currently, this workflow is only supported for the following Operating systems:
+- Ubuntu >= 16.04
+- CentOS >= 7
+- Fedora >= 24
+- MacOSX
+
+## Minimum Requirements
+Minimum resource requirements for a single node BigchainDB dev setup. **The more the better**:
+- Memory >= 512MB
+- VCPUs >= 1
+
+## Download the scripts
+```text
```

Comment:
These are instructions for developers, some of whom may be working on BigchainDB Server code, so they may already have local recent versions of `stack.sh` and `unstack.sh`. Maybe add a sentence before these instructions saying:

> If you're working on BigchainDB Server code, on a branch based on recent code, then you already have local recent versions of `stack.sh` and `unstack.sh` in your `bigchaindb/pkg/scripts/` directory. Otherwise you can get them using:

Also, you might help them by saying they can use `master` in place of `${GIT_BRANCH}` to get the scripts from the current master branch.

Reply:
I will add a note along these lines. Default is master.